src/defaults.js

```diff
@@ -364,6 +364,12 @@ module.exports = {
      * @instance
      */
     fixedColAlign: 'center',
+    /**
+     * @default
+     * @type {string}
+     * @instance
+     */
+    defaultColAlign: 'center',
```

Comment:
I would go with simply `halign` but I see you're trying to parallel `fixedColAlign` so I could live with `colAlign` or `dataColAlign`... but really I prefer `halign` which is already a familiar prop in getCell config.